ppl/zqd/space/space.go

```diff
@@ -126,6 +126,7 @@ func loadSpaces(ctx context.Context, p iosrc.URI, conf config, logger *zap.Logge
 		if err != nil {
 			return nil, err
 		}
+		m.alphaFileMigrator.Add(store)
```

Comment:
If we are going to migrations this way (which I think is a good idea), why not just get rid of the migration check in filestore.Open and have it run the search against alphazng? Sure the search will be slow but at least they'll be able to access their space while the migration happens.

Reply:
I did consider this, but on balance decided against. Reading from the alpha zng files via the tzng bridge is quite slow, and the existing seek index mechanism won't work with them. 

With these & the changes in #1574, a Brim user who happens to run a search on a space will get the usual UI spinner, so it looks like their initial search is just taking a long time if they get to a space before the backgrounder gets to it.